data/json/items/book/martial.json

```diff
@@ -104,6 +104,15 @@
     "description": "A complete guide to Shōtōkan Karate.",
     "martial_art": "style_karate"
   },
+  {
+    "id": "manual_kickboxing",
+    "copy-from": "book_martial",
+    "type": "BOOK",
+    "name": { "str": "The Ultimate Guide to Kickboxing", "str_pl": "copies of Ultimate Guide to Kickboxing" },
+    "price_postapoc": 2000,
+    "description": "The Ultimate Guide to learning and mastering Kickboxing.  From beginner to black belt",
```

Comment:
```suggestion
    "description": "The ultimate guide to learning and mastering kickboxing, from beginner to black belt.",
```

Reply:
Alright, I'll make that change in about two hours from now. Gotta go right now. Thanks